content/pages/whoweare.html

```diff
@@ -140,6 +140,24 @@ <h3 id="vadim_melnik_">Vadim Melnik</h3>
     </div>
   </div>
 
+  <div class="row">
+    <h3 id="cody_baker_">Cody Baker</h3>
+    <figure><img src="/theme/img/team/cody.jpg" class="pull-left"/></figure>
+    <div class="col-md-4 col-sm-4 position">Research Software Engineer</div>
+    <div class="col-md-5 col-sm-5 contact">
+      <a href="mailto:cody.c.baker.phd@gmail.com?Subject=Hello Cody @CON" class="email">E-Mail</a>
+      <a href="codycbakerphd.com" class="home">Home Page</a>
```

Comment:
```suggestion
      <a href="https://codycbakerphd.com" class="home">Home Page</a>
```
Acutally i think this will fix-- looks like its being treated as an internal link without the protocol

Reply:
As it is now works fine and passes CI, but either way seems fine